src/main/kotlin/com/workos/directorysync/models/PreviousAttributes.kt

```diff
@@ -0,0 +1,16 @@
+package com.workos.directorysync.models
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * An object representing the previous attributes of names and values
+ * that have changed and their previous values.
+ *
+ * @param previousAttributes The object containing the previous keys and values for changes.
+ */
+data class PreviousAttributes @JsonCreator constructor(
```

Comment:
unused-- trying to see how we can compose `User` and `UserWithPreviousAttributes`

Reply:
This is not as straightforward as I thought. Kotlin doesn't allow you to inherit constructors. Removing for now and will figure it out on the next iteration.

Reply:
will work on this in the next iteration